src/main/java/backend/Teamoverview/TeamOverview.kt

```diff
@@ -1,179 +1,100 @@
-package backend.Teamoverview
-
-import backend.model.BasicEntity
-import backend.model.event.Event
-import backend.model.event.Team
-import backend.model.location.Location
-import com.fasterxml.jackson.core.JsonGenerator
-import com.fasterxml.jackson.databind.SerializerProvider
-import com.fasterxml.jackson.databind.annotation.JsonSerialize
-import com.fasterxml.jackson.databind.ser.std.StdSerializer
+package backend.teamoverview
+
+import backend.model.misc.Coord
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
 import java.time.LocalDateTime
 import java.time.ZoneId
-import javax.persistence.*
-
-@Entity
-class TeamOverview : BasicEntity {
 
-    private constructor() : super()
+interface TeamOverview {
 
-    constructor(team: Team, event: Event) {
-        setOrUpdateValues(event, team)
-    }
+    interface Event {
+        val id: Long
 
-    fun setOrUpdateValues(event: Event, team: Team) {
-        this.teamId = team.id!!
-        this.teamName = team.name
-        this.event = Event(event.id, event.title)
-        this.members = team.members.map {
-            TeamMember(it.id, it.firstname, it.lastname, it.emergencynumber, it.phonenumber)
-        }
+        @Value("#{target.title}")
```

Comment:
what is target?

Reply:
`target` is a proxy to access the values of the entity directly from the query. We only have access to beans and managed properties